specification/security/resource-manager/Microsoft.Security/preview/2023-08-01-preview/examples/Pricings/GetPricingByNameCloudPosture_example.json

```diff
@@ -0,0 +1,41 @@
+{
+  "parameters": {
+    "api-version": "2023-08-01-preview",
+    "scopeId": "subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "pricingName": "CloudPosture"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/providers/Microsoft.Security/pricings/CloudPosture",
+        "name": "CloudPosture",
+        "type": "Microsoft.Security/pricings",
+        "properties": {
+          "pricingTier": "Standard",
+          "isEnabled": true,
+          "freeTrialRemainingTime": "PT0S",
+          "enablementTime": "2023-03-01T12:42:42.1921106Z",
+          "enforce": false,
+          "extensions": [
+            {
+              "name": "AgentlessVmScanning",
```

Comment:
This extensio has also additional properties, please update the example:
            {
                "name": "AgentlessVmScanning",
                "isEnabled": "True",
                "additionalExtensionProperties": {
                    "ExclusionTags": "[{\"Key\":\"D1\",\"Value\":\"R1\"},{\"Key\":\"Daniel1\",\"Value\":\"Test1\"}]"
                }
            }

Reply:
Update also in all relevant example files (List DCSPM & Servers)